tests/trainer/test_trainer.py

```diff
@@ -1623,9 +1623,10 @@ def is_any_loss_nan_or_inf(log_history):
         self.assertFalse(is_any_loss_nan_or_inf(log_history_filter))
 
     def test_train_and_eval_dataloaders(self):
-        if torch_device in ["cuda", "xpu"]:
+        if torch_device in ["cuda"]:
```

Comment:
I am ok, but since you added that before, it was a mistake back then?

Reply:
yes, it was a mistake by me. In that time I just tested in a single card, so may missed. In my recent test in multi-card env, i find it fail and debug to the DP issue... That's why i put a comment in `else` statement, to remind me and other xpu guys to avoid later mistakes in case we forgot the history, :)